docs/api-reference/globe-view.md

```diff
@@ -0,0 +1,58 @@
+# GlobeView Class (Experimental)
+
+The [`GlobeView`] class is a subclass of [View](/docs/api-reference/view.md). This view projects the earth into a 3D globe.
```

Comment:
My understanding is that this is a limited 3D globe view, in that it always looks at the center of the earth. 

While very impressive and valuable for many cases, this is not a general 3D globe view allowing us to e.g. fly over the planet seeing the bend of the horizon or match the capabilities of systems like Google Earth or Cesium.

If so, I think this should be stated clearly in the doc, its purpose is for showing the entire globe on center, mainly for a full-world view.